css/civicrm.icons.css

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file
+ * Styling for CiviCRM toolbar icons.
+ */
+
+/* Bar Icons */
+.toolbar-bar .toolbar-icon-civicrm:before {
+  background-image: url("../icons/bebebe/CiviCRM.svg");
+}
+.no-svg .toolbar-bar .toolbar-icon-civicrm:before {
+  background-image: url("../icons/bebebe/CiviCRM.png");
+}
+.toolbar-bar .toolbar-icon-civicrm:active:before,
+.toolbar-bar .toolbar-icon-civicrm.is-active:before {
+  background-image: url("../icons/ffffff/CiviCRM.svg");
+}
+.no-svg .toolbar-bar .toolbar-icon-civicrm:active:before,
+.no-svg .toolbar-bar .toolbar-icon-civicrm.active:before {
+  background-image: url("../icons/ffffff/CiviCRM.png");
+}
+  
+  
```

Comment:
There seems to be a trailing whitespace on these two lines.

Reply:
Fixed.